crate/pages/blog/crate-service-discovery-dns-aws.html

```diff
@@ -0,0 +1,56 @@
+title: Crate adds Service Discovery via DNS and the AWS EC2 API
```

Comment:
Service discovery = discovery via DNS SRV records

More correct then: Crate adds (Unicast) Host Discovery via DNS and ...